### DISCUSSION TITLE
Deployment in Production 404 missing old files in users browsers

### DISCUSSION THREAD

**Discussion by 1cor13** [nuxt/nuxt#27692] [Questions]
Hello, whenever I deploy in production, the users get many 404 errors because the old hashed files are removed from `_nuxt` folder. What does nuxt recommend in dealing with these issues.
![image](https://github.com/nuxt/nuxt/assets/60481125/81493c19-f2f8-4c61-aca7-553e8f42d434)

**Comment by TheAlexLichter** [comment]
This is expected if you don't store/cache the old files, e.g. via CDN. Nuxt should be able to reload and serve new files though. See also https://github.com/nuxt/nuxt/pull/19038 and following
